netshare/learn/setup_cross_chunks_data.py

```diff
@@ -102,26 +107,39 @@ def get_word2vec_model(df: pd.DataFrame) -> Optional[Word2Vec]:
 
     if not word2vec_cols:
         logger.info("Skipping word2vec embedding: no word2vec columns")
-        return None
+        return None, None
 
     if word2vec_config["word2vec"]["pretrain_model_path"]:
         logger.info("Word2vec: Loading pretrained model")
-        return Word2Vec.load(word2vec_config["word2vec"]["pretrain_model_path"])
+        return (
+            Word2Vec.load(word2vec_config["word2vec"]["pretrain_model_path"]),
+            word2vec_cols,
+        )
 
     logger.info("Word2vec: training model")
     os.makedirs(get_preprocessed_data_folder(), exist_ok=True)
-    word2vec_model_path = word2vec_train(
+    word2vec_train(
         df=df,
-        out_dir=get_word2vec_model_directory(),
         model_name=word2vec_config["word2vec"]["model_name"],
         word2vec_cols=word2vec_cols,
         word2vec_size=word2vec_config["word2vec"]["vec_size"],
         annoy_n_trees=word2vec_config["word2vec"]["annoy_n_trees"],
     )
-    return Word2Vec.load(word2vec_model_path)
+    word2vec_model_path = get_word2vec_model_path()
```

Comment:
I thought that `build_annoy_dictionary_word2vec` will call this function itself, and not get it as an input,
But that's really not mandatory